code/modules/nanogate/nanogate_items.dm

```diff
@@ -3,16 +3,16 @@
 	name = "nanite suit control module"
 	desc = "A lighter, less armoured rig suit made from nanites and attached to someone's spine."
 	icon_state = "ihs_rig_old"
-	suit_type = "nanite suit"
+	suit_type = "nanite"
 	armor_list = list(
-		melee = 30,
-		bullet = 30,
-		energy = 30,
+		melee = 35,
+		bullet = 35,
+		energy = 35,
 		bomb = 25,
 		bio = 100,
 		rad = 100
 	)
```

Comment:
```suggestion
	)
	seal_delay = 20
```
I gave all ''light'' class RIGs a seal_delay buff awhile back. This amount is the same as the light and EVA rig This should have gotten it back then really mostly forgot since its hiding in its own file.